run_tests.sh

```diff
@@ -1,3 +1,4 @@
+pytest tests/test_neutronics_utils.py -v --cov=paramak --cov-report term --cov-report xml
 pytest tests/test_utils.py -v --cov=paramak --cov-report term --cov-report xml
```

Comment:
```suggestion
pytest tests/test_utils.py -v --cov=paramak --cov-append --cov-report term --cov-report xml
```

`--cov-append` flag missing here